framework/db/ActiveQuery.php

```diff
@@ -81,6 +81,11 @@ class ActiveQuery extends Query implements ActiveQueryInterface
     const EVENT_INIT = 'init';
 
     /**
+     * string placeholder to be replaced by the table alias of a query.
+     */
+    const ALIAS_PLACEHOLDER = '@alias';
```

Comment:
what? once alias for all queries ?
try it:
A::find()->joinWth(['b' => function($q){$q->andWhere(['@alias.id' => 1])}])->andWhere(['@alias.id' => 2])